README.md

```diff
@@ -24,3 +24,5 @@ pip install flake8-trio
 - **TRIO101** `yield` inside a nursery or cancel scope is only safe when implementing a context manager - otherwise, it breaks exception handling.
 - **TRIO102** it's unsafe to await inside `finally:` unless you use a shielded
   cancel scope with a timeout"
+- **TRIO103** `except BaseException` and `except trio.Cancelled` with a code path that doesn't re-raise
```

Comment:
For the readme, where we can be more verbose, I think it's important to explain that we ignore any raises in loops: "Note  that any `raise` statement in a loop is ignored, as it's difficult to rule out the zero-iterations case".

Reply:
It's actually `break` that's the biggest offender, the zero-iteration case (or `continue`s) can be covered if there's an `else` that always raises. So in theory it could accept loops that have no `break`s, an `else`, and both bodies always `raise` - but that's silly.

Fixed :+1: